app/translate.py

```diff
@@ -105,7 +115,7 @@ def _stream_translate(import_id: str, source: IO, dest: IO, file_type: str, tran
 
         dest.write(chunk.encode())  # encodes as utf-8 by default
 
-def validate_import_url(import_url: Optional[str], user_info: UserInfo) -> bool:
+def validate_import_url(import_url: Optional[str], import_filetype: Optional[str], user_info: UserInfo) -> bool:
```

Comment:
I'm not well versed in the type hinting library we use and it caused me some grief last time I was in this codebase so I might be biased, but should the import_url and import_filetype params be marked optional if we throw exceptions when they're None?  How about?
```suggestion
def validate_import_url(import_url: str, import_filetype: str, user_info: UserInfo) -> bool:
```

Reply:
hm, good question, I was just following the convention here. In the case where one of these values is actually None, I do like that we have full control over the log message and error sent to the user; that's nice.

That said, the first-pass validation actually happens in the model itself: https://github.com/broadinstitute/import-service/blob/1f617b7785fc2d1ac43575802a4426e27bd742df/app/server/routes.py#L33
so these error message should never be seen in practice